contributing/telemetry.md

```diff
@@ -6,6 +6,8 @@ Similar to what vscode does, we are reporting on certain events that happen in t
 
 All extensions leverage the telemetryService in `salesforce-vscode-core` extension in order to send data. The service in `salesforce-vscode-core` is resposible of showing an information message to users and properly initializing the service for other extensions in this repository to leverage. In the same line as vscode, we provide the user an opt out mechanism. We check that user settings `telemetry.enableTelemetry` and `salesforcedx-vscode-core.telemetry.enabled` are enabled before initializing the service.
 
+Additionally, if telemetry is disabled for the Salesforce CLI via the `disableTelemetry` configuration setting, or the `SFDX_DISABLE_TELEMETRY` environment variable, the telemetryService will not send data.
```

Comment:
@ekapner Can you review these changes, please.

Reply:
Although not highlighted in green: The service in `salesforce-vscode-core` is **responsible**(typo) of showing an...

Instead of: In the same line as vscode,
Please change: Similar to VSCode, ...

Edits for green/highlighted text:
Additionally, if telemetry is disabled for Salesforce CLI using the `disableTelemetry` configuration setting or the `SFDX_DISABLE_TELEMETRY` environment variable, the telemetryService doesn't send data.